beta-20161006.md

```diff
@@ -0,0 +1,29 @@
+---
+title: What's New in beta-20161006
+toc: false
+summary: Additions and changes in CockroachDB version beta-20161006.
+---
+
+## Oct 6, 2016
+
+Although this week's release contains only a few internal changes and doc updates, note that we are preparing to merge numerous features from our `develop` branch to `master` for inclusion in an upcoming release.  
+
+### Binaries
+
+- [Mac](https://binaries.cockroachdb.com/cockroach-beta-20161006.darwin-10.9-amd64.tgz)
+- [Linux](https://binaries.cockroachdb.com/cockroach-beta-20161006.linux-amd64.tgz)
+
+### Internal Changes
+
+- Nodes now periodically exchange their configured maximum clock offset and exit fatally if they find another node operating with a different configuration. [#9612](https://github.com/cockroachdb/cockroach/pull/9612)
+- Internal low-level instrumentation has been added for debugging performance bottlenecks. [#9638](https://github.com/cockroachdb/cockroach/pull/9638)
```

Comment:
Without these, the notes would be nothing but doc updates. Plus, we've featured internal changes is a handful of previous releases.